INSTALL.md

```diff
@@ -1,46 +1,116 @@
-- `GOVDELIVERY_BASE_URL`
-- `GOVDELIVERY_ACCOUNT_CODE` (GovDelivery account variable)
-- `GOVDELIVERY_USER` (GovDelivery account variable)
-- `GOVDELIVERY_PASSWORD` (GovDelivery account variable)
-- `SUBSCRIPTION_SUCCESS_URL` (Forwarding location on Subscription Success)
+**NOTE:** If you use ZSH you'll need to use
+[zsh-autoenv](https://github.com/Tarrasch/zsh-autoenv), but we can't provide support for issues that may arise.
 
-> You can also export the above environment variables to your `.bash_profile`,
-or use your favorite alternative method of setting environment variables.
+### Elasticsearch
 
-**NOTE:** GovDelivery is a third-party web service that powers our subscription forms.
-Users may decide to swap this tool out for another third-party service.
-The application will function but throw an error if the above GovDelivery values are not set.
+[Install Elasticsearch](http://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html)
+however you'd like. (We use [Homebrew](http://brew.sh)):
+```bash
+brew install elasticsearch
+```
+
+Just as with autoenv, Homebrew will output similar instructions after installation:
+```bash
+To have launchd start elasticsearch at login:
+    ln -sfv /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/*.plist ~/Library/LaunchAgents
+Then to load elasticsearch now:
+    launchctl load ~/Library/LaunchAgents/homebrew.mxcl.elasticsearch.plist
+Or, if you don't want/need launchctl, you can just run:
+    elasticsearch --config=/Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/config/elasticsearch.yml
+```
+
+Any time you work on the project, you'll need to open a new tab and run that last line.
+If you'll be working on the project consistently, we suggest using the first option
+utilizing `launchd`.
+
+If you need to find this info again later, you can run:
+```bash
+brew info elasticsearch
+```
+
+### Sheer
+
+To [install sheer](https://github.com/cfpb/sheer#installation), start by cloning the
+Sheer GitHub project to wherever you keep your projects (not inside cfgov-refresh):
+```bash
+git clone https://github.com/cfpb/sheer.git
+```
+
+Create a virtualenv for Sheer, which you'll name `cfgov-refresh`:
+```bash
+mkvirtualenv cfgov-refresh
+```
+
+The new virtualenv will activate right away. To activate it later on
+(say, in a new terminal session) use the command `workon cfgov-refresh`.
+You'll know you have a virtual environment activated if you see the name of it in
+parentheses before your terminal prompt. Ex:
+```bash
+(cfgov-refresh)username$
+```
 
+Install Sheer into the virtualenv with the `-e` flag (which allows you to make changes to
+Sheer itself). The path to Sheer is the root directory of the GitHub repository you
+cloned earlier, which will likely be `../sheer`:
+```bash
+pip install -e ~/path/to/sheer
+```
 
-# 2. Front-end setup
+Install Sheer's Python requirements:
+```bash
+pip install -r ~/path/to/sheer/requirements.txt
+```
+
+### GovDelivery
+
```

Comment:
Maybe add a note here about what GovDelivery is needed for. A blurb like https://github.com/cfpb/cfgov-refresh/pull/482/files#diff-7d442b7eb49f5fc377f51e74b291cfc1R158